add version 0.3.17 to r-inline

glennpj opened the pull request.

glennpj committed:
add version 0.3.17 to r-inline

adamjstewart commented on var/spack/repos/builtin/packages/r-inline/package.py:
```diff
@@ -7,14 +7,17 @@
+    version('0.3.17', sha256='792857b2ebd408d6523424d2f6bb7297e241d4b28ab32372f6a9240c8cd554f3')
     version('0.3.15', sha256='ff043fe13c1991a3b285bed256ff4a9c0ba10bee764225a34b285875b7d69c68')
     version('0.3.14', sha256='fd34d6bf965148d26d983a022a0ff7bc1a5831f6ca066deee3f6139894dfc931')
 
```
This version requirement doesn't seem necessary

glennpj replied:
It was specified in the 0.3.14 and the 0.3.15 versions and dropped in the 0.3.17 version. That version of R is so old though that it makes sense to drop it from the spack recipe as well.

glennpj committed:
Drop R version constraint

A really old version of R was specified in the 0.3.14 and 0.3.15
versions of r-inline. This constraint was dropped in the 0.3.17 version.
Drop it from the spack recipe as well.

adamjstewart commented:
Not sure why this PR is having so many problems with its unit tests. I would keep closing and reopening.
